### PR TITLE
Add v1.4.1 of mattermost-plugin-zoom to Marketplace

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
This PR adds v1.4.1 of Zoom to marketplace

#### Ticket Link
None

### CODE REVIEW

**Comment by larkox** [comment]
@jfrerich For some reason when I try to run "make generate", the statik file does not get updated correctly, creating this problem on the CI. Would you mind running yourself the "make generate" and see if it passes the CI?

File: plugins.json [hunk 1]
```diff
@@ -4195,6 +4195,234 @@
...
+    "download_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/download/v1.4.1/zoom-1.4.1.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.4.1",
+    "signature": "",
+    "manifest": {
+      "id": "zoom",
+      "name": "Zoom",
+      "description": "Zoom audio and video conferencing plugin for Mattermost 5.2+.",
```
Comment (aaronrothschild) [inline plugins.json:4207]:
```suggestion
      "description": "Zoom audio and video conferencing plugin for Mattermost 5.12+",
```

**Comment by jfrerich** [comment]
@larkox, I pushed a commit with the results of `make generate`.

Question: Why is `v1.4.0` included in this release?  Was it not included before?

**Comment by larkox** [comment]
The previous release was not released to the marketplace because it had a bug that we didn't want to show on the marketplace. Not sure if we should remove it from the marketplace, and if there is any "good way" to do that. Right now, if we were not to add it, we would have to remove it any time we do `make plugin.json` in the future.

**Comment by jfrerich** [comment]
Thanks for the clarification.  I'm not sure we even allow getting a specific version from the marketplace.  Don't we only show the latest version anyway and in that case, we should be ok.

File: plugins.json (hunk 1 above)
Comment (larkox) [reply plugins.json:4207]: This is something that comes from the manifest, and it is not easily changed (we would need to cut a new release if I am not mistaken). Nevertheless, this is just on the description, so should not give big problems.

**Comment by larkox** [comment]
I just realized that the signature for this release was never generated. I will look into the problem.

**Comment by DHaussermann** [comment]
@larkox and @jfrerich when I deploy this to my local market place server. I see that `v1.4.1` is available. But, the upgrade does not work.
I see `Unable to verify plugin signature.` when I click Update. 
Screen Shot 2020-07-17 at 9 48 15 AM

**Comment by DHaussermann** [comment]
Sorry @larkox I did not read the last comment before posting 🤦

**Comment by jfrerich** [comment]
@larkox, if you need an assistance, feel free to DM and zoom me!

**Comment by larkox** [comment]
@hanzei Can you take a look at this? For some reason, when I do `make generate` it does not create the right file. If you could create the right file and push the changes, I would really appreciate that.

**Comment by hanzei** [comment]
@DHaussermann @iomodo @jfrerich @larkox I've run `make generate` to fix the build issue. LTAL

**Comment by larkox** [comment]
@DHaussermann This should be ready to QA now.

**Pull request merged**